### PR TITLE
print schemas when describing plan in svcat

### PR DESCRIPTION
**Pull request by kibbles-n-bytes**
Prints the schemas for the plan when you run `svcat describe plan ${PLANNAME}` in YAML format. This way you don't need to muck around with kubectl and our human-unfriendly resource names to find the schemas.

closes: #1720

### CODE REVIEW

File: cmd/svcat/testdata/responses/clusterserviceplans.json [hunk 1]
```diff
@@ -44,7 +44,31 @@
...
+	  "properties": {
+	    "testInstanceProperty": {
+	      "description": "A test instance property.",
+	      "type": "string"
+	    }
+	  },
+	  "required": [
+	    "testInstanceProperty"
+	  ],
+	  "type": "object"
```
Comment (carolynvs) [inline cmd/svcat/testdata/responses/clusterserviceplans.json]: I'm unfamiliar with the schema properties. What does the `type` field in this example refer to? `properties`?

File: cmd/svcat/testdata/output/describe-plan-with-schemas.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  UUID:          cc0d7529-18e8-416d-8946-6f7456acd589  
+  Status:        Active                                
+  Free:          false                                 
+  Class:         user-provided-service                 
+
+Instances:
+No instances defined
+
+Instance Create Schema:
+properties:
```
Comment (carolynvs) [inline cmd/svcat/testdata/output/describe-plan-with-schemas.txt]: Would it be possible to indent the schema yaml? It isn't critical but would be easier to read. What do you think?

```console
$ svcat describe plan premium
  Name:          premium                               
  Description:   Premium plan                          
  UUID:          cc0d7529-18e8-416d-8946-6f7456acd589  
  Status:        Active                                
  Free:          false                                 
  Class:         user-provided-service                 

Instances:
No instances defined

Instance Create Schema:
  properties:
    testInstanceProperty:
      description: A test instance property.
      type: string
  required:
  - testInstanceProperty
  type: object

Binding Create Schema:
  properties:
    testBindingProperty:
      description: A test binding property.
      type: string
  required:
  - testBindingProperty
  type: object
```

Comment (kibbles-n-bytes) [reply cmd/svcat/testdata/output/describe-plan-with-schemas.txt]: Done.

File: cmd/svcat/testdata/responses/clusterserviceplans.json (hunk 1 above)
Comment (kibbles-n-bytes) [reply cmd/svcat/testdata/responses/clusterserviceplans.json]: It's saying that the `parameters` field in the instance/binding request payload is of type `object`, and the `properties` are the key+values expected.

**Comment by carolynvs** [comment]
@arschles What's the use case for adding a flag to describe to hide the schemas?

**Comment by arschles** [comment]
@carolynvs it'd be nice just to let folks suppress extra output if they want. don't really know if anyone would use it, but imo it'd be nice to have

**Comment by MHBauer** [comment]
I'd guess because the schemas can be large, but I've only heard that, I don't actually have any.

**Comment by kibbles-n-bytes** [comment]
Yeah, schemas can get large, so I'm fine with adding a flag to suppress that output. @arschles done.

**Pull request merged**